BEProjectsApp/models.py

```diff
@@ -3,7 +3,7 @@
 from django.contrib.auth.models import User
 
 DOMAIN_CHOICES = [
-    ("1", ("Data Mining & Analytics")),
+    ("Data Mining & Analytics", ("Data Mining & Analytics")),
```

Comment:
Why is this different then others?

Reply:
due to the previous one the data which was stored in database was numbers as string now it will be actual data

Reply:
latest commit is perfect